src/tools/curve_tool_define.cpp

```diff
@@ -37,3 +37,4 @@ DEFINE_string(snapshotCloneDummyPort, "8081", "dummy port of snapshot clone, "
                                     "if specify several, the order should "
                                     "be the same as snapshot clone addr");
 DEFINE_uint64(chunkSize, 16777216, "chunk size");
+DEFINE_bool(useChunkFilePoolAsWalPool, true, "use chunkfilepool as walpool");
```

Comment:
It is inconvenient to use GFlag and be determined by humans，you can read chunkserver's config metric and choose to expose walpool or not.

Reply:
> It is inconvenient to use GFlag and be determined by humans，you can read chunkserver's config metric and choose to expose walpool or not.

done